tests/dive_n7.bme

```diff
@@ -0,0 +1,4522 @@
+#12501:00009Q9Q00000000
+#12501:000000009R9R0000
+#12501:0000000000009S9S
+#12501:0000747400000000
+#12501:000000006J6J0000
+#12501:0000000000006S6S
+#12501:00
+#12501:6W6W000000000000
+#12504:1100000000000000000012000000000000000013000000000000000014000000000000000000150000000000000000001600000000000000000017000000000000000000180000000000000000001900000000000000001A00000000000000001B000000000000001C000000000000001D00000000000000001E0000000000000000001F00000000000000001G000000000000001H0000000000000000001I000000000000001J000000000000001K000000000000001L000000000000000000
+#12511:1F
+#12512:0011001100110011
+#12514:3R
+#12515:00400000
+#12516:1O1O
+#12518:003G
+#12519:0000003O
+
+#12601:00
+#12601:00
+#12601:0000000000000011
+#12601:00
+#12601:00
+#12601:00
+#12601:00
+#12601:00
+#12601:00001G1G1F001G1G1F001G1G00000000
+#12601:1L1L
+#12601:1S1S1S1S1S1S1S1S
+#12601:00
+#12601:2Z252Q30
+#12601:00250000002Q00000030000000260000
+#12601:302Q26302R26312R
+#12601:00260030002R00260031002R0027002W
+#12601:002R00310027002S
+#12601:00000031000000270000002S0000002X
+#12601:00
+#12601:00
+#12601:9M9M000000000000
+#12601:00009N9N00000000
+#12601:000000009P9P0000
+#12601:0000000000009T9T
+#12604:0100000000000000000000030000000000000000000005000000000000000000000700000000000000000000090000001W00000000001V00000000001U00000000000000000000000000000000000000000000001T00000000001S00000000001M00000000001N00000000001O00000000001P000000000000000000000000001O00000000001N000000001M000000002200000000002100000000002000000000000000000000000000000000000000000000001Z00000000001Y0000000000
+#12611:1F00001F
+#12612:0011001100110000
+#12613:3Z0000000000000000000000001G0000
+#12614:003F0000
+#12615:000000003S00001F
+#12616:1O1O
+#12618:0000003W
+#12619:0000000000000000000000000000001G
+
+#12701:00
+#12701:0011001100110011
+#12701:0000000000000000000000000000000000000000001G001G0000000000000000
+#12701:00
+#12701:41
+#12701:003H0000
+#12701:003U
+#12701:00000042
+#12701:00
+#12701:1L1L1L1L
+#12701:1S1S1S1S1S1S1S1S
+#12701:00
+#12701:00
+#12701:31272S32
+#12701:00270000002S00000032000000280000
+#12701:302S26322R28312T
+#12701:00260032002R00280031002T00270032
+#12701:002R00310027002S
+#12701:00000031000000270000002S00000031
+#12701:00
+#12701:9Q9Q000000000000
+#12701:00009R9R00000000
+#12701:000000009U9U0000
+#12701:0000000000009V9V
+#12704:0100000000050000000009000000000D000000000H0000002800002700002600000000000000000000002500002400000L00002B00000X00002D00001600002F00000100002H00001N00002J00000X00002L00001600002N00000100002O0000
+#12711:1F1F1F1F
+#12713:001G0000001G0000001G0000001G0000
+#12715:001F001F001G001G
+#12716:1O1P1O1P
+#12719:0000001G0000001G0000001G0000001G
+
+#12801:0F
+#12801:00
+#12801:1V
+#12801:00
+#12801:1E
+#12801:00
+#12801:3D
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:00
+#12801:9W
+#12804:34
+#12811:1F
+#12816:1N
+
```

Comment:
It seems that this is an original one created by xi. Did you obtain permission to redistribute this file? All other BMS files uploaded here were created by me. I want to be clean about copyright, so you ought to obtain permission for commiting it.

Or instead you can add a test case about BMS that contians only `#WAV` and `#BMP` to resolve it.

Reply:
I've never considered about it, thanks for noticing me so much, I will create a test document instead of uploading the original one.